CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -506,6 +506,21 @@ private static void OnIsBrowserInitializedChanged(DependencyObject d, Dependency
 
         protected virtual void OnIsBrowserInitializedChanged(bool oldValue, bool newValue)
         {
+            managedCefBrowserAdapter.GetZoomLevelAsync()
+                .ContinueWith(new Action<Task<double>>((Task<double> previous) =>
+                    {
+                        if (previous.IsCompleted)
+                        {
+                            UiThreadRunAsync(() =>
+                            {
+                                SetValue(ZoomLevelProperty, previous.Result);
```

Comment:
Or something else now or in the future. I didn't mean to be literal, it is just defensive coding.